tests/Arch/RoutingTest.php

```
@@ -0,0 +1,104 @@
+ */
+function applicationRoutes(): array
+{
+    return array_values(array_filter(
+        RouteFacade::getRoutes()->getRoutes(),
+        fn (Route $route): bool => ! Str::startsWith($route->uri(), ['_debugbar', 'storage/']),
+    ));
+}
+
+function isPublicRoute(Route $route): bool
+{
+    return Str::startsWith($route->uri(), ArchTestCase::except('routing.public-endpoints')) || $route->uri() === '/';
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method 'isPublicRoute'. (undefined)

(StaticAccess)